HelpMyStreetFE/HelpMyStreetFE/Views/Shared/Components/JobStatusChangePopup/Admin_MarkAsOpenPopupFromAppliedFor.cshtml

```diff
@@ -0,0 +1,25 @@
+﻿@model HelpMyStreetFE.Models.Account.Jobs.JobStatusChangePopupViewModel;
+@{
+    var isShift = Model.RequestType == HelpMyStreet.Utils.Enums.RequestType.Shift;
+    var requestType = isShift ? "shift" : "request";
+}
+<div class="popup__content">
+    <partial name="_PopupCloseButton" />
+    <div class="popup__content__wrapper">
+        <div class="popup__content__header">Mark as Open?</div>
+        <div class="popup__content__text">
+            <p>Have you decided that the volunteer who applied to help with this @requestType is unable to complete it?</p>
+            <p>We can return it to the “Open @(isShift ? "Shifts" : "Requests")” list so that other volunteers can accept it.</p>
+            <p>
```

Comment:
Maybe add a sentence "The volunteer will receive a notification to say that their application has not been successful."